tests/mod.test.ts

```diff
@@ -146,10 +146,10 @@ Deno.test({
   async fn() {
     await editChannel(data.channelID, {
       name: "edited-channel",
-      overwrites: [
+      permissionOverwrites: [
         {
           id: data.roleID,
-          type: OverwriteType.ROLE,
+          type: "ROLE",
```

Comment:
Breaking change lol

Reply:
Can we leave this change? It took a long time to make this. A longer to revert.